images/tensorflow-notebook/cuda12/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+ARG REGISTRY=quay.io
+ARG OWNER=jupyter
+ARG BASE_CONTAINER=$REGISTRY/$OWNER/scipy-notebook
+FROM $BASE_CONTAINER
+
+LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
+
+# Fix: https://github.com/hadolint/hadolint/wiki/DL4006
+# Fix: https://github.com/koalaman/shellcheck/wiki/SC3014
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Install CUDA libs and cuDNN with mamba
+RUN mamba install --yes -c nvidia/label/cuda-12.3.2 \
+    'cuda-libraries-dev' \
+    'cuda-nvcc' \
+    'cuda-nvtx' \
+    'cuda-cupti' \
+    'cudnn' && \
+    mamba clean --all -f -y && \
+    fix-permissions "${CONDA_DIR}" && \
+    fix-permissions "/home/${NB_USER}"
+
+ENV XLA_FLAGS="--xla_gpu_cuda_data_dir=${CONDA_DIR}" \
```

Comment:
Could you please add a link where this flag is recommended?

Reply:
I just tried and is seems obsolete now. I got it e. g. from [here](https://github.com/tensorflow/tensorflow/issues/23783#issuecomment-462771434) or [here](https://stackoverflow.com/a/64872998/2414411). It was related to a warning about not finding `ptxas` and maybe also `libdevice.10.bc`. Then XLA might not have worked (`jit_compile=True`). It seems it got fixed in a recent version. I will remove the line.